### PR TITLE
Rebranding appstudion->konflux

### PR DESCRIPTION
**Pull request by skabashnyuk**
Rebranding messages 
 - appstudion->konflux
 - rhtap -> konflux

### CODE REVIEW

File: controllers/component_build_controller_pac.go [hunk 1]
```diff
@@ -985,13 +986,13 @@ func (r *ComponentBuildReconciler) ConfigureRepositoryForPaC(ctx context.Context
...
+		CommitMessage:  "Konflux update " + component.Name,
 		BranchName:     generateMergeRequestSourceBranch(component),
 		BaseBranchName: baseBranch,
-		Title:          "Appstudio update " + component.Name,
+		Title:          "Konflux update " + component.Name,
 		Text:           mergeRequestDescription,
-		AuthorName:     "redhat-appstudio",
-		AuthorEmail:    "rhtap@redhat.com",
+		AuthorName:     "konflux",
+		AuthorEmail:    "konflux@konflux-ci.com",
```
Comment (mmorhun) [inline controllers/component_build_controller_pac.go:995]: Maybe we should have `no-replay.*`?

File: controllers/component_build_controller_pac.go [hunk 2]
```diff
@@ -985,13 +986,13 @@ func (r *ComponentBuildReconciler) ConfigureRepositoryForPaC(ctx context.Context
...
-		CommitMessage:  "Appstudio update " + component.Name,
+		CommitMessage:  "Konflux update " + component.Name,
 		BranchName:     generateMergeRequestSourceBranch(component),
 		BaseBranchName: baseBranch,
-		Title:          "Appstudio update " + component.Name,
+		Title:          "Konflux update " + component.Name,
 		Text:           mergeRequestDescription,
-		AuthorName:     "redhat-appstudio",
-		AuthorEmail:    "rhtap@redhat.com",
+		AuthorName:     "konflux",
```
Comment (mmorhun) [inline controllers/component_build_controller_pac.go:994]: Changing commiter will prevent Renovate from update existing PRs. Please add previous renovate as known commiter.

**Review by mmorhun — APPROVED** [review]

**Review by mmorhun — APPROVED** [review]

File: controllers/component_build_controller_pac.go [hunk 3]
```diff
@@ -985,13 +986,13 @@ func (r *ComponentBuildReconciler) ConfigureRepositoryForPaC(ctx context.Context
...
+		AuthorEmail:    "konflux@no-replay.konflux-ci.com",
```
Comment (ralphbean) [inline controllers/component_build_controller_pac.go:995]:
```suggestion
		AuthorEmail:    "konflux@no-reply.konflux-ci.dev",
```

File: controllers/component_build_controller_pac.go [hunk 4]
```diff
@@ -1095,13 +1096,13 @@ func (r *ComponentBuildReconciler) UnconfigureRepositoryForPaC(ctx context.Conte
...
+			CommitMessage:  "Konflux purge " + component.Name,
+			BranchName:     "konflux-purge-" + component.Name,
 			BaseBranchName: baseBranch,
-			Title:          "Appstudio purge " + component.Name,
+			Title:          "Konflux purge " + component.Name,
 			Text:           "Pipelines as Code configuration removal",
-			AuthorName:     "redhat-appstudio",
-			AuthorEmail:    "rhtap@redhat.com",
+			AuthorName:     "konflux",
+			AuthorEmail:    "konflux@no-replay.konflux-ci.com",
```
Comment (ralphbean) [inline controllers/component_build_controller_pac.go:1105]:
```suggestion
			AuthorEmail:    "konflux@no-reply.konflux-ci.dev",
```

File: pkg/git/github/github_helper_debug_test.go [hunk 5]
```diff
@@ -93,13 +93,13 @@ func TestCreatePaCPullRequest(t *testing.T) {
...
+		CommitMessage:  "Konflux update " + componentName,
+		BranchName:     "konflux-" + componentName,
 		BaseBranchName: "",
-		Title:          "Appstudio update " + componentName,
+		Title:          "Konflux update " + componentName,
 		Text:           "Pipelines as Code configuration proposal",
-		AuthorName:     "redhat-appstudio",
-		AuthorEmail:    "rhtap@redhat.com",
+		AuthorName:     "konflux",
+		AuthorEmail:    "konflux@no-replay.konflux-ci.com",
```
Comment (ralphbean) [inline pkg/git/github/github_helper_debug_test.go:102]:
```suggestion
		AuthorEmail:    "konflux@no-reply.konflux-ci.dev",
```

File: pkg/git/github/github_helper_debug_test.go [hunk 6]
```diff
@@ -123,13 +123,13 @@ func TestUndoPaCPullRequest(t *testing.T) {
...
+		CommitMessage:  "Konflux purge " + componentName,
+		BranchName:     "konflux-purge-" + componentName,
 		BaseBranchName: "",
-		Title:          "Appstudio purge " + componentName,
+		Title:          "Konflux purge " + componentName,
 		Text:           "Pipelines as Code configuration removal",
-		AuthorName:     "redhat-appstudio",
-		AuthorEmail:    "rhtap@redhat.com",
+		AuthorName:     "konflux",
+		AuthorEmail:    "konflux@no-replay.konflux-ci.com",
```
Comment (ralphbean) [inline pkg/git/github/github_helper_debug_test.go:132]:
```suggestion
		AuthorEmail:    "konflux@no-reply.konflux-ci.dev",
```

File: pkg/git/gitlab/gitlab_helper_debug_test.go [hunk 7]
```diff
@@ -53,13 +53,13 @@ func TestEnsurePaCMergeRequest(t *testing.T) {
...
+		CommitMessage:  "Konflux update " + componentName,
+		BranchName:     "konflux-" + componentName,
 		BaseBranchName: "",
-		Title:          "Appstudio update " + componentName,
+		Title:          "Konflux update " + componentName,
 		Text:           "Pipelines as Code configuration proposal",
-		AuthorName:     "redhat-appstudio",
-		AuthorEmail:    "rhtap@redhat.com",
+		AuthorName:     "konflux",
+		AuthorEmail:    "konflux@no-replay.konflux-ci.com",
```
Comment (ralphbean) [inline pkg/git/gitlab/gitlab_helper_debug_test.go:62]:
```suggestion
		AuthorEmail:    "konflux@no-reply.konflux-ci.dev",
```

File: pkg/git/gitlab/gitlab_helper_debug_test.go [hunk 8]
```diff
@@ -92,13 +92,13 @@ func TestUndoPaCMergeRequest(t *testing.T) {
...
+		CommitMessage:  "Konflux purge " + componentName,
+		BranchName:     "konflux-purge-" + componentName,
 		BaseBranchName: "",
-		Title:          "Appstudio purge " + componentName,
+		Title:          "Konflux purge " + componentName,
 		Text:           "Pipelines as Code configuration removal",
-		AuthorName:     "redhat-appstudio",
-		AuthorEmail:    "rhtap@redhat.com",
+		AuthorName:     "konflux",
+		AuthorEmail:    "konflux@no-replay.konflux-ci.com",
```
Comment (ralphbean) [inline pkg/git/gitlab/gitlab_helper_debug_test.go:101]:
```suggestion
		AuthorEmail:    "konflux@no-reply.konflux-ci.dev",
```

**Pull request merged**